Using as a dependency in an oclif plugin errors out

Using as a dependency in an oclif plugin errors out as: 
`Cannot find module '@airtasker/spot' or its corresponding type **declarations.**`
on command invocation. 


Steps to reproduce the behavior:
1.Create an oclif plugin 
2. Add @airtasker/spot as a dependency 
3. Use `createProjectFromExistingSourceFile` or `validateProject`
4. Add the plugin in another oclif project

--- a/lib/src/spec-helpers/helper.ts
+++ b/lib/src/spec-helpers/helper.ts
@@ -1,3 +1,4 @@
+import * as path from "path";
 import { Project, SourceFile, ts } from "ts-morph";
 
 /**
@@ -34,7 +35,7 @@ interface FileDetail {
 }
 
 /**
- * Create an AST project with the `@airtasker/spot` depedency loaded.
+ * Create an AST project with the `@airtasker/spot` dependency loaded.
  */
 export function createProject(): Project {
   return new Project({
@@ -54,7 +55,7 @@ export function createProject(): Project {
       experimentalDecorators: true,
       baseUrl: "./",
       paths: {
-        "@airtasker/spot": ["./lib/src/lib"]
+        "@airtasker/spot": [path.join(__dirname, "../lib")]
       }
     }
   });

